pallene/syntax_errors.lua

```diff
@@ -175,6 +175,8 @@ local errors = {
     AssignNotToVar = "Expected a valid lvalue in the left side of assignment but found a regular expression.",
 
     CastMissingType = "Expected a type for the cast expression.",
+
+    LocalOrExportRequired = "Function definition must include 'local' or 'export' modifier."
```

Comment:
Is this related to #261 or is it something else?

Reply:
No, it's not related to #261. I thought it would be nice to have a custom message is all. I've removed it.